### PR TITLE
fix: specify correct type version

### PR DESCRIPTION
**Pull request by snqb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "@types/rc-slider": "^8.6.6",
     "@types/react": "^16.14.0",
     "@types/react-datepicker": "^3.1.8",
-    "@types/react-dom": "^16.14.0",
```
Comment (snqb) [inline package.json:46]: this version apparently doesn't exist :D
`"^16.9.14"` is the latest before 17

File: yarn.lock [hunk 2]
```diff
@@ -2431,9 +2431,9 @@ __metadata:
     "@tanem/react-nprogress": ^4.0.8
     "@types/node": ^12.12.38
     "@types/rc-slider": ^8.6.6
-    "@types/react": ^16.14.23
+    "@types/react": ^16.14.0
     "@types/react-datepicker": ^3.1.8
-    "@types/react-dom": ^16.9.14
+    "@types/react-dom": ^16.9.7
```
Comment (esonhey) [inline yarn.lock]: shouldn't be 16.14.0?

Comment (snqb) [reply yarn.lock]: https://github.com/logicalclocks/quartz/pull/215#discussion_r826102847

**Review by esonhey — APPROVED** [review]
Good to go

**Pull request merged**